remotestore/remotestore.go

```diff
@@ -597,16 +597,16 @@ func addBlocksToRemoteStoreIndex(
 	return addToRemoteStoreIndex(ctx, blobClient, addedStoreIndex)
 }
 
-func onPreflighMessage(
+func onPreflightMessage(
 	s *remoteStore,
 	storeIndex longtaillib.Longtail_StoreIndex,
 	message preflightGetMessage,
 	prefetchBlockMessages chan<- prefetchBlockMessage) {
-	const fname = "onPreflighMessage"
+	const fname = "onPreflightMessage"
 	log := logrus.WithFields(logrus.Fields{
-		"fname":   fname,
-		"s":       s,
-		"message": message,
+		"fname":            fname,
+		"s":                s,
+		"len(blockHashes)": message.blockHashes,
```

Comment:
Shouldn’t this be len(message.blockHashes) here?

Reply:
You were too quick. It's why I left it in draft. 😄